x/mint/keeper/msg_server_test.go

```diff
@@ -24,9 +24,39 @@ func (s *IntegrationTestSuite) TestUpdateParams() {
 			request: &types.MsgUpdateParams{
 				Authority: s.mintKeeper.GetAuthority(),
 				Params: types.Params{
-					MintDenom:             sdk.DefaultBondDenom,
-					AnnualReductionFactor: sdk.NewDecWithPrec(-13, 2),
-					BlocksPerYear:         uint64(60 * 60 * 8766 / 5),
+					MintDenom:          sdk.DefaultBondDenom,
+					BondingAdjustment:  sdk.NewDecWithPrec(-13, 2),
+					TargetBondingRatio: sdk.NewDecWithPrec(25, 2),
+					InflationCoef:      sdk.NewDecWithPrec(73, 2),
+					BlocksPerYear:      uint64(60 * 60 * 8766 / 5),
+				},
+			},
+			expectErr: true,
+		},
+		{
+			name: "set invalid params",
```

Comment:
Would be good to elaborate the test case names to give more context to what is being tested 🙂  At least to differentiate them from each other

Reply:
You're right it's better ;)